core/src/display_object/movie_clip.rs

```diff
@@ -218,6 +218,10 @@ pub struct MovieClipData<'gc> {
 
     /// Show a hand cursor when the clip is in button mode.
     avm2_use_hand_cursor: Cell<bool>,
+
+    /// If this movie is currently executing an ImportAssets/2.
+    /// If true, this movie should **not** execute, and should be considered as still loading.
+    awaiting_import: Cell<bool>,
```

Comment:
Nit: I think it would make more sense to put this in the `PreloadProgress` struct.

Reply:
Good idea, pushed